src/main.rs

```diff
@@ -59,32 +62,32 @@ fn parse_matches() -> clap::ArgMatches {
             Arg::with_name("slots_per_epoch")
                 .long("slots-per-epoch")
                 .takes_value(true)
-                .help("override the number of slots in an epoch"),
+                .help("override the number of slots in an epoch. Default for cluster_type: development -> 8192.
+                Default for cluster_type: devnet, testnet, mainnet-beta -> 432000 (1 epoch every ~= 2 days)"),
         )
         .arg(
             Arg::with_name("target_lamports_per_signature")
                 .long("target-lamports-per-signature")
                 .takes_value(true)
-                .help("Genesis config. target lamports per signature"),
+                .help("Genesis config. target lamports per signature. Default: 10000"),
```

Comment:
Gosh, I'm learning a lot through these PRs. I can't believe this concept still exists!